src/renderer/containers/query-browser.jsx

```diff
@@ -22,10 +22,11 @@ require('../components/react-tabs.scss');
 
 
 const STYLES = {
-  wrapper: { paddingTop: '50px' },
-  container: { display: 'flex', padding: '10px 10px 50px 10px' },
-  sidebar: { overflowY: 'auto', height: '100vh' },
-  content: { flex: 1, overflow: 'auto', height: '100vh' },
+  wrapper: {  },
+  container: { display: 'flex', height: '100vh', box-sizing: 'border-box', padding: '50px 10px 40px 10px'  },
```

Comment:
`box-sizing` is not a valid react style attribute https://facebook.github.io/react/tips/inline-styles.html